content/en/blog/_posts/2023-11-xx-scheduler-queueinghint/index.md

```diff
@@ -0,0 +1,99 @@
+---
+layout: blog
+title: "Kubernetes v1.28: QueueingHint brings a new possibility to optimize our scheduling"
```

Comment:
1.29 now, please. Also - until we change the style guide: [Title Case](https://en.wikipedia.org/wiki/Title_case).

Reply:
Good to know.

Reply:
I've marked this conversation as “not resolved” because the feedback is still applicable.